docs/operations.rst

```diff
@@ -51,3 +51,5 @@ The important bits here are:
 Backups
 *******
 
+Performing regular backups of the state store is an important component for any production deployment of ``alchemiscale``.
+To 
```

Comment:
Missing some intended content here @dotsdl

Reply:
Ah haha, thank you!

Reply:
Think I've got this resolved! Thanks @hmacdope!